docs/references/calfits_memo.tex

```diff
@@ -12,177 +13,155 @@
 \end{itemize}
 
-There are also some optionally required parameters that depend on the
-calibration type. These parameters include.
+	\subsubsection{Required if CALSTYLE = ``sky''}
+	\begin{itemize}
+	\item{\textbf{CATALOG}: \emph{string} (Required if CALSTYLE = ``sky''.) Name of the calibration catalog. (sky\_catalog)}
+	\item{\textbf{FIELD}: \emph{string} (Required if CALSTYLE = ``sky''.) A short string describing the field center or dominant source. (sky\_field)}
+	\item{\textbf{REFANT}: \emph{string} (Required if CALSTYLE = ``sky''.) Phase reference antenna. (ref\_antenna\_name)}
+	\end{itemize}
+	
+	
+	\subsubsection{Required if CALTYPE = ``delay''}
+	\begin{itemize}
+	\item{\textbf{FRQRANGE:} \emph{float} Required if CALTYPE = ``delay''. Frequency range that solutions are valid for, in Hz. (freq\_range)}
+	\end{itemize}
+	
+	
+\subsection{Optional Keywords}
 \begin{itemize}
-\item{\textbf{delay\_array}: Required if cal\_type =``delay''. Array of delays with
-    units of seconds. Shape: (Nants\_data, Nspws, 1, Ntimes, Njones), type = float.}
-\item{\textbf{gain\_array}: Required if cal\_type = ``gain''. Array of gains, 
-    shape: (Nants\_data, Nspws, Nfreqs, Ntimes, Njones), type = complex float.}
-\item{\textbf{freq\_range}: Required if cal\_type = ``delay''. Frequency range that
-   solutions are valid for. list: [start\_frequency, end\_frequency] in Hz.}
+\item{\textbf{BL\_RANGE:} \emph{float} Range of baselines used for calibration. (baseline\_range)}
+\item{\textbf{DIFFUSE:} \emph{string} Name of diffuse model used for sky model. (diffuse\_model)}
+\item{\textbf{GNSCALE:} \emph{string} The gain scale of the calibration, which indicates the units of the calibrated visibilities. For example, Jy or K. (gain\_scale)}
+\item{\textbf{HASHCAL:} \emph{string} Commit hash of calibration software (from ORIGCAL) used to generate solutions. (git\_hash\_cal)}
+\item{\textbf{NSOURCES:} \emph{integer} Number of sources used in sky model. (Nsources)}
+\item{\textbf{ORIGCAL:} \emph{string} Origin (on github for example) of calibration software. URL and branch. (git\_origin\_cal)}
 \end{itemize}
 
-In addition to the required parameters, there are a number of truly optional
-parameters that may be passed in. These include:
+\section{Antenna HDU}
+This HDU is a binary table extension with the extension name ``ANTENNA''.
+The Antennas HDU is mandatory in all \emph{calfits} files, and stores detailed information about the calibration solution, per antenna.
+This binary table has a number of entries equaling the number of antennas in the dataset, and three fields, containing the individual antennas' names, indices, and integer antenna numbers matching the 0th axis of the uvcal object ``gain\_array.''
```

Comment:
maybe change "entries" to "rows"? I find that a little clearer for some reason.

Reply:
I still see "entries" here, did you decide that made more sense than "rows"?

Reply:
whoops, I guess that one didn't get stashed. I'll fix it back again!